backends/imgui_impl_wgpu.cpp

```diff
@@ -769,6 +830,153 @@ void ImGui_ImplWGPU_NewFrame()
         ImGui_ImplWGPU_CreateDeviceObjects();
 }
 
+//--------------------------------------------------------------------------------------------------------
+// MULTI-VIEWPORT / PLATFORM INTERFACE SUPPORT
+// This is an _advanced_ and _optional_ feature, allowing the backend to create and handle multiple viewports simultaneously.
+// If you are new to dear imgui or creating a new binding for dear imgui, it is recommended that you completely ignore this section first..
+//--------------------------------------------------------------------------------------------------------
+
+
+static void ImGui_ImplWGPU_CreateWindow(ImGuiViewport* viewport)
```

Comment:
This should probably be moved out of the backend and user should provide this instead - there's no easy way to handle this for every platform/WebGPU impl.

Reply:
100% this is the main thing I was struggling with trying to fix, would it be just a callback that a user would need to assign to ?